index.html

```diff
@@ -59,5 +59,163 @@ <h3>LET'S CONNECT</h3>
+          <button class="see-project">See Project</button>
+        </article>
+      </div>
+      <div class="cards card-4">
+        <div class="main-img">
+          <img
+            src="./img/Snapshoot Portfolio (3).png"
+            alt="art printing web page"
+          />
+        </div>
+        <article class="second-container">
+          <h2>Tonic</h2>
+          <div class="horizontal-section">
+            <span>CANOPY</span>
+            <input type="radio" checked />Back End Dev
+            <input type="radio" checked />2015
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="skills">
+            <div>html</div>
+            <div>css</div>
+            <div>javascript</div>
+          </div>
+          <button class="see-project">See Project</button>
+        </article>
+      </div>
+      <!-- About section -->
+      <section class="about">
+        <h2>
+          About <br />
+          Myself
+        </h2>
+        <p>
+          Hello I'm a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don't hestiate to
+          contact me.
+        </p>
+        <h3>LET'S CONNECT</h3>
+        <ul class="socials">
+          <li>
+            <a href="https://github.com/J-C-S-V"
+              ><img src="./img/github.svg" alt="twitter logo"
+            /></a>
+          </li>
+          <li>
+            <a
+              href="https://www.linkedin.com/in/juan-carlos-sanchez-vargas-a308b014b/"
+              ><img src="./img/Linkedin icon.svg" alt="linkedin logo"
+            /></a>
+          </li>
+          <li>
+            <a href="https://angel.co/"
+              ><img src="./img/angellist.svg" alt="angel logo"
+            /></a>
+          </li>
+          <li>
+            <a href="https://twitter.com/home"
+              ><img src="./img/twitter.svg" alt="twitter logo"
+            /></a>
+          </li>
+          <li>
+            <a href="https://medium.com/"
+              ><img src="./img/medium.svg" alt="medium logo"
+            /></a>
+          </li>
+        </ul>
+        <button class="see-resume">Get my resume</button>
+        <div class="languages-main-container">
+          <h4 class="languages">Languages <span>˅</span></h4>
+          <div class="skills-boxed">
+            <img src="/img/JS.png" alt="Javascript" />JavaScript
+          </div>
+          <div class="skills-boxed">
+            <img src="/img/HTML.png" alt="HTML" />HTML
+          </div>
+          <div class="skills-boxed">
+            <img src="/img/CSS3.png" alt="CSS3" />CSS
+          </div>
+          <div class="frameworks-skills">Frameworks <span>></span></div>
+          <hr />
+          <div class="frameworks-skills">Skills<span>></span></div>
```

Comment:
- [ ] In this part of your project, according to the project requirements you are to use the (`ul` ad `li` tags).
![image](https://user-images.githubusercontent.com/102699958/219553132-cb487331-7a32-4739-9e39-f92c89831438.png)
- [ ] So please fix this as well and ensure that the `Languages ` container is vertically aligned  with the `Frameworks` and `Skills` and have the right styles applied as per the Figma Design :+1: 
- [ ] The icons are also not well styled at the moment so please style them according to the Figma design :100: 
- Your Design 
![image](https://user-images.githubusercontent.com/102699958/219552993-8d1066a4-43bc-4808-b532-20199f4ac336.png)
 - The Figma Design
 
![image](https://user-images.githubusercontent.com/102699958/219553790-a7842317-2000-4d04-97b8-ebc284aa9fcc.png)

Reply:
Hello @NahnahAJ,

Could you please go deeply on this suggestion? 'The icons are also not well styled at the moment so please style them according to the Figma design', It is too general and I don't understand what is wrong with the svg icons.

Thanks for your help!